sources/extension-models/timestamp/TimeStampVersion.xsd

```diff
@@ -0,0 +1,232 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<xsd:schema xmlns:ext="http://standards.ieee.org/downloads/11073/11073-10207-2017/extension"
+            xmlns:sdpi="urn:oid:1.3.6.1.4.1.19376.1.6.2.10.1.1.1"
+            xmlns:pm="http://standards.ieee.org/downloads/11073/11073-10207-2017/participant"
+            attributeFormDefault="unqualified"
+            elementFormDefault="qualified"
+            targetNamespace="urn:oid:1.3.6.1.4.1.19376.1.6.2.10.1.1.1"
+            xml:lang="en"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <xsd:import schemaLocation="../ExtensionPoint.xsd"
+              namespace="http://standards.ieee.org/downloads/11073/11073-10207-2017/extension" />
+  <xsd:import schemaLocation="../BICEPS_ParticipantModel.xsd"
+              namespace="http://standards.ieee.org/downloads/11073/11073-10207-2017/participant" />
+  <xsd:simpleType name="EpochVersion">
+    <xsd:annotation>
+      <xsd:documentation>Time-stamp epoch version. The default version for any timestamp not versioned is the current epoch version. </xsd:documentation>
+    </xsd:annotation>
+    <xsd:restriction base="xsd:nonNegativeInteger" />
+  </xsd:simpleType>
+  <!---->
+  <!---->
+  <!--Epochs-->
+  <xsd:element name="Epochs">
+    <xsd:annotation>
+      <xsd:documentation>An extension to version epochs arising from non-slewing time adjustments.
+
+This extension can be attached to the pm:ClockState/ext:Extension element.
+		</xsd:documentation>
+    </xsd:annotation>
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="sdpi:EpochsType">
+          <xsd:attribute name="Version"
+                         default="0">
+            <xsd:annotation>
+              <xsd:documentation>Current epoch version.</xsd:documentation>
+            </xsd:annotation>
+            <xsd:simpleType>
+              <xsd:restriction base="sdpi:EpochVersion">
+                <xsd:minInclusive value="0" />
+              </xsd:restriction>
+            </xsd:simpleType>
+          </xsd:attribute>
+        </xsd:extension>
+      </xsd:complexContent>
+    </xsd:complexType>
+  </xsd:element>
+  <xsd:complexType name="EpochsType">
+    <xsd:annotation>
+      <xsd:documentation>Collection of timesteps between epochs. Must include every version referenced elsewhere in the MDIB.</xsd:documentation>
+    </xsd:annotation>
+    <xsd:sequence>
+      <xsd:element name="Epoch"
+                   type="sdpi:EpochType"
+                   minOccurs="0"
+                   maxOccurs="unbounded" />
+    </xsd:sequence>
+  </xsd:complexType>
+  <xsd:complexType name="EpochType">
+    <xsd:annotation>
+      <xsd:documentation>
+Type defining step change to a new Epochs.
+
+Defines the from the previous time-frame reference to the next time-frame reference. Adding this offset to this Timestamp gives the point in time (to an unbiased external observer) when this non-slewing time-step occurred in the next epoch's time-frame reference.
```

Comment:
Something's missing here. Duration, maybe?

Reply:
Edited for clarity:
> Type defining a transition between epochs.
> Defines the step-change, which occurs at a single point in time, from the previous time-reference frame to the next time-reference frame. Adding this Offset to this Timestamp gives the point in time (to an unbiased external observer)  when this time-step occurred using the clock from the next epoch's time-reference frame.
> For example, if device time advanced by 1 hour in epoch 0 at 10 am, there will be an Epoch entry for epoch version 0 with a timestamp of 10am and Offset of +1 hour. The equivalent time in epoch version 1 will be 11 am.